REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,131 @@
+
+## Status
+
+There are four statuses available to indicate the maturity, reliability, and potential usability of each repository within the Cilium project: ```STABLE```, ```INCUBATING```, ```SANDBOX```, or ```DEPRECATED```. 
+
+Repositories of type ```Ecosystem``` or ```Infra``` can have any status, while repositories of the ```Core``` type must have a ```STABLE``` status. Status does not apply to ```Special``` repositories. 
+
+The status of a repository is subject to change over time as it increases or decreases in activity. See the section on [Change of scope or status](#change-of-scope-or-status) for more detail.
+
+### ```STABLE```
+
+Stable repositories are those with the highest degree of maturity and reliability, where significant changes or updates are less frequent. They are actively being maintained by Cilium committers, and are intended for use by adoptors.
+
+Criteria for ```STABLE``` status include:
+* Actively supported by both the committers and larger Cilium community
+* Regularly updated and released
+* Documentation is provided and kept up-to-date
+* Automated testing via unit, functional, and integration test suites on multiple platforms (when applicable)
+* Pre-built artifacts are published by an automated system for multiple platforms (when applicable)
+* Documented release process (when applicable)
+
+### ```INCUBATING```
+
+Incubating repositories are those in an intermediate stage of maturity, and are more subject to significant changes and updates as they continue to grow. They may be used by adoptors with the caution that their support may vary and they are generally not appropriate for critical use cases.
+
+Criteria for ```INCUBATING``` status include:
+* Supported by at least two committers
+* Basic documentation is provided
+* Minimal CI/CD pipelines are implemented (when applicable)
+
+### ```SANDBOX```
+
+Sandbox repositories are in the earliest stage of maturity. They are not recommended to be used in production, but could be used by those who want to experiment with new features. Very little to no support is provided.
+
+### ```DEPRECATED```
+
+Deprecated repositories are those that are no longer maintained and are only kept for historical purposes. They are not recommended for use by adoptors. They are typically [archived](#archival) and read-only.
+
+## Lifecycle
+
+### Addition
+
+Repositories of type ```Ecosystem```, ```Infra```, or ```Special``` can be added into the GitHub organization at the discretion of the Cilium committers (in other words, by lazy consensus).
+
+Any new repository of any type must be added into the appropriate list in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md) and be assigned a status (if applicable).
+
+#### Adding ```Core``` repositories
+
+Given their importance to the project, repositories of the ```Core``` type must be added through a more formal process:
```

Comment:
since they are end user facing, they could potentially change the scope of the project which is why it needs to be a bit more formal